cartridge/webui/api-auth.lua

```diff
@@ -75,8 +75,21 @@ end
 local function get_auth_params()
     local callbacks = auth.get_callbacks()
     local params = auth.get_params()
+
+    local user, username
+    local session_username = auth.get_session_username()
+    if session_username ~= nil then
+        user = auth.get_user(session_username)
+    end
+
+    if user ~= nil and user.fullname ~= nil then
```

Comment:
I think we should prefer user.username instead of session_username

Reply:
It was done as it was decided here - https://github.com/tarantool/cartridge/issues/726#issuecomment-607333734

Also seems that "session_username" and "user.username" are identical things, however session_username could be easier extracted from session rather than username from user object.

Reply:
It's OK. `user.username == get_session_username()`, `user.fullname == tdg.username`, it's what we discussed yesterday.